common/src/main/java/com/github/khanshoaib3/minecraft_access/features/inventory_controls/GroupGenerator.java

```diff
@@ -467,8 +466,51 @@ else if (screen.getHandler() instanceof HopperScreenHandler)
             foundGroups.add(itemOutputGroup);
         }
 
-        if (unknownGroup.slotItems.size() > 0) {
-            foundGroups.add(unknownGroup);
+        // Adjacency is used here instead of slot.inventory because some mods have non-adjacent slots in the same
```

Comment:
It seems like these lines need indentation?

Reply:
I was using neovim to make some quick edits this morning and for whatever ungodly reason it insisted on using tabs for indentation, I though I got all of them but there are probably a few lines that I didn't catch.

Reply:
Those particular lines do look indented properly though and I made them in idea, I've fixed the one line that Neovim nicely stuffed for me.

Reply:
Yeah, I'm wrong, sorry.